spec/models/defect_spec.rb

```diff
@@ -55,6 +55,13 @@
     end
   end
 
+  describe 'added_at' do
+    it 'automatically populates added_at' do
+      defect = create(:defect)
+      expect(defect.reload.added_at).not_to be_nil
```

Comment:
Let's also make sure it's actually the creation date, not the string `"CURRENT_TIMESTAMP"`. I've had issues doing this in the past.

Reply:
Thanks @erbridge! I've made this test take into account time. Unfortunately, freezing doesn't really work in this setup because Postgres is dealing with the CURRENT_TIMESTAMP, so I've limited the result to only care about the date portion of the output, which should vastly reduce the room for error.

If you've approached this testing in a different way in another project, I'd love to learn from it.